README.md

```diff
@@ -10,8 +14,53 @@
 
 
 ## Usage
-* Tests must end in `_(spec|test).(coffee|js)`
-* Run tests with ```mycha run```
+
+* `mycha install`
+  * prompts you for where you would like to install your default test helper, `testHelperPath`
+
+  * runs
+    ```
+    npm install --save-dev chai
+    npm install --save-dev sinon
+    npm install --save-dev sinon-chai
+    ```
+
+  * writes to `testHelperPath`
+    ```coffee
+    chai = require 'chai'
+    sinon = require 'sinon'
+    chai.use require 'sinon-chai'
+
+    global.expect = chai.expect
+    global.sinon = sinon
+
+    process.env.NODE_ENV = 'test'
+    ```
+
+  * writes to `mycha.coffee`
+    ```coffee
+    module.exports =
+
+      # Default options to pass to mocha (can be overriden by command line options)
+      mochaOptions:
+        colors: yes
+        compilers: 'coffee:coffee-script/register'
+        reporter: 'dot'
+
+      # Regular expressions for files that are tests
+      testFiles: [
```

Comment:
Only make this an array if there are multiple elements in here. For now this can just be

```coffeescript
testFiles: /_(spec|test)\\.(coffee|js)$/
```